src/main.js

```diff
@@ -68,3 +68,9 @@ new Vue({
     })
   }
 }).$mount('#app')
+// e2e tests use the offline mode, so here we expose the Vue.js app so Cypress can access it programmatically
+// e.g. window.app.$store and window.app.$workflowService.
+// Ref: https://www.cypress.io/blog/2017/11/28/testing-vue-web-application-with-vuex-data-store-and-rest-backend/
+if (['test', 'offline'].includes(process.env.NODE_ENV)) {
+  window.app = app
+}
```

Comment:
Same change in #454 . So once either this or that PR is merged, I will fix conflicts in the other PR :+1: